app/views/simple_smart_answers/_current_question.html.erb

```diff
@@ -11,17 +11,11 @@
           <p class="error-message" role="alert">Please answer this question</p>
         <% end %>
         <%= hidden_field_tag 'response', '' %>
-        <ul class="options">
```

Comment:
Is this class used elsewhere or can it be removed from the CSS?

Reply:
No, it doesn't seem to be used here or in static. The `selectable` class [comes from static](https://github.com/alphagov/static/blob/9e8795b262c948a65b798029864cbe04f193262a/app/assets/stylesheets/helpers/_selectable.scss).